lib/server.js

```diff
@@ -21,35 +21,37 @@ module.exports = {
   serverRoutes: {},
   serverPolicies: {},
   serverHandlers: {},
+  webConfig: {},
 
   createServer (app) {
     const config = app.config
     const server = express()
-    this.port = config.web.port
-    this.host = config.web.host
-    this.ssl = config.web.ssl
-    this.redirectToHttps = config.web.redirectToHttps || false
+    this.webConfig = _.cloneDeep(app.config.web)
```

Comment:
Looks good to me; in fact, I think your solution is better than what I did for trailpack-hapi. I moved some of the setup logic in configure, but this would restrict the ability of other trailpacks to extend the config object in `configure()`, since hapi might configure() itself before they have a chance to run.